src/containers/forms/PractitionerForm/UserIdSelect/index.tsx

```diff
@@ -102,10 +102,18 @@ export const UserIdSelect: React.FC<Props> = props => {
     setOpenMRSUsers(unMatchedUsers);
   };
 
-  const options = openMRSUsers.map((user: OpenMRSUser) => ({
-    label: user.display,
-    value: user.uuid,
-  }));
+  const options = React.useMemo(() => {
+    return openMRSUsers
+      .map((user: OpenMRSUser) => ({
+        label: user.display,
+        value: user.uuid,
+      }))
+      .sort((userA, userB) => {
+        const userALabel = userA.label;
+        const userBLabel = userB.label;
+        return userALabel === userBLabel ? 0 : userALabel > userBLabel ? 1 : -1;
+      });
+  }, [openMRSUsers]);
```

Comment:
Is there a test that covers this?

Reply:
Added one.

Reply:
@p-netm  where? (I am assuming we are not using this branch for anything)